src/main/java/io/xstefank/wildlfy/bot/format/checks/BaseBranchCheck.java

```diff
@@ -0,0 +1,43 @@
+package io.xstefank.wildlfy.bot.format.checks;
+
+import io.xstefank.wildlfy.bot.config.RegexDefinition;
+import org.kohsuke.github.GHPullRequest;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class BaseBranchCheck implements Check {
```

Comment:
I don't think we want to restrict the base branch. Can you point me where did you find this in Tyr?

Reply:
it has been modified from [this one](https://github.com/jboss/tyr/blob/main/tyr-core/src/main/java/org/jboss/tyr/check/AddTargetBranchLabelCheck.java) thought it might be a good addition since there are cases where the maintainer might want the PRs to go to a branch other than main. I can also add this check from tyr.

Reply:
But that check is doing something else right? It's adding a label, not marking format as invalid.